views/index.hbs

```diff
@@ -8,23 +8,23 @@
 	</button>
 </form>
 {{#if entries}}
-	<ul class="entry-list">
-		{{#each entries}}
-			<li class="entry">
-				<h2 class="entry__question">
-					{{this.question}}
-				</h2>
-				<div class="entry__answer">
-					{{{this.answer}}}
-				</div>
-				<div class="entry__help">
-					{{this.helpful}}/?? people found this helpful. Was this entry helpful?
-					<button class="button button--small">Yes</button>
-					<button class="button button--small">No</button>
-				</div>
-			</li>
-		{{/each}}
-	</ul>
+<ul class="entry-list">
+	{{#each entries}}
+	<li class="entry">
+		<h2 class="entry__question">
+			{{this.question}}
+		</h2>
+		<div class="entry__answer">
+			{{{this.answer}}}
+		</div>
+		<div class="entry__help">
+			<span id="displayCount">{{this.helpful}}</span>/<span id="totalNumber"></span> people found this helpful. Was this entry helpful?
+			<button id="Button1" class="button button--small">Yes</button>
+			<button id="Button2" class="button button--small">No</button>
```

Comment:
`Button1` and `Button2` are bad names for the buttons. Consider using a more descriptive name like `button-helpful` and `button-not-helpful`.